package.json

```diff
@@ -10,7 +10,7 @@
   "license": "GPL-3.0",
   "keywords": [],
   "scripts": {
-    "test": "mocha build/src/**/*.test.js",
+    "test": "npm run pretest && mocha build/src/**/*.test.js",
```

Comment:
I actually learned this today: NPM has a few built-in hooks that are automatically run, including [`pretest` and `posttest`](https://docs.npmjs.com/misc/scripts), which are run automatically before and after the `test` command.

However, I agree that explicit >= implicit. Would it be clearer to change the names of the `pretest` and `posttest` commands to be `pre-test` and `post-test`, just so that they're not run automatically?

Reply:
Yeah, that sounds reasonable to me. It’s cool that NPM does that! Though that trick won’t work with other package managers, like `yarn`.

Reply:
ping @SaltyQuetzals how do you want to move forward with this?

Reply:
Mb, totally forgot about this.

I think this is fine as-is, It's nbd for unit tests to run twice.